03-Mains/Heaving_Water_mass_transformation_decomposition.ipynb

```diff
@@ -0,0 +1,2108 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    plt.gca().invert_yaxis()
</pre><p>doesn't <code>plt.gca()</code> just return the same thing as <code>ax</code>?</p><p>Also, suggest using <code>ax1</code>,<code>ax2</code>,and <code>ax3</code> here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>